src/cat.js

```diff
@@ -32,12 +32,9 @@ function _cat(options, files) {
     if (!fs.existsSync(file))
       common.error('no such file or directory: ' + file);
 
-    cat += fs.readFileSync(file, 'utf8') + '\n';
+    cat += fs.readFileSync(file, 'utf8');
   });
 
-  if (cat[cat.length-1] === '\n')
```

Comment:
@nfischer: Whoops... I apparently can't read... Never mind. (I thought you added this).